src/main/java/taxi/dao/CarDaoImpl.java

```diff
@@ -211,9 +211,13 @@ private Driver parseDriverFromResultSet(ResultSet resultSet) throws SQLException
         Long driverId = resultSet.getObject("id", Long.class);
         String name = resultSet.getString("name");
         String licenseNumber = resultSet.getString("license_number");
+        String login = resultSet.getString("login");
+        String password = resultSet.getString("password");
```

Comment:
You are trying to get the 'password' field from the result set, but 'password' field is not included in SQL query. You need to add 'password' in your SQL query.